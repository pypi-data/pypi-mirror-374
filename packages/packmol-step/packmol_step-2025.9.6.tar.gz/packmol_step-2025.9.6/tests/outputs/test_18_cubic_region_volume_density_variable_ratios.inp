tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside cube 0.0 0.0 0.0 21.5443
   number 41
end structure
structure input_2.pdb
   inside cube 0.0 0.0 0.0 21.5443
   number 123
end structure
