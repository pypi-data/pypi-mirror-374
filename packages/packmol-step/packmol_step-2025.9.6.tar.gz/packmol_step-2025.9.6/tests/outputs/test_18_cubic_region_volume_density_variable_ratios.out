    Will create a cubic region containing the following molecules:

        +-------------+-------------+---------+
        |  Component  | Structure   |   Ratio |
        |-------------+-------------+---------|
        |    fluid    | c1ccccc1    |       1 |
        |    fluid    | O           |       3 |
        +-------------+-------------+---------+

    The dimensions of the region will be calculated from the volume 10.0 nm³.
    The number of molecules of the fluid will be obtained  by using the density
    0.9 g/ml.

Created a cubic region 21.54 Å on a side with 164 fluid molecules

    +-------------+-------------+---------------+----------+------------+
    |  Component  | Structure   |   Requested % |   Number |   Actual % |
    |-------------+-------------+---------------+----------+------------|
    |    fluid    | c1ccccc1    |            25 |       41 |         25 |
    |    fluid    | O           |            75 |      123 |         75 |
    +-------------+-------------+---------------+----------+------------+

There are a total of 861 atoms in the cell giving a density of 0.89976 g/ml.